endpoints/api/billing.py

```diff
@@ -989,9 +996,16 @@ def post(self, orgname):
 
                 user_subscription_ids = [int(subscription["id"]) for subscription in subscriptions]
                 if int(subscription_id) in user_subscription_ids:
+                    quantity = 1
```

Comment:
I'm assuming this means that if a subscription is returned from the marketplace, there's at least one of it.
Is there a case where this could be 0?

Reply:
I don't think so, a quantity of 0 should just mean the subscription is terminated anyways I would imagine